### ISSUE TITLE
Xunit plugin stdout printed three times

### ISSUE THREAD

**Issue by lodegaard** [nose-devs/nose#868]
When I use the Xunit plugin in my tests the captured standard output is printed three times. First as the message attribute of <failure>, then as a part of the <failure> body and finally as <system-out>.
It seems this occures because the capture-plugin appends the captured stdout to the err object when a failure is handled.

Are anyone else experiencing this? If so I can provide a fix for this issue

**Comment by jszakmeister** [comment]
Yes, I see it, but is it really a problem?  Part of the issue is that many different test tools choose to show different parts of the xml in various views, so I imagine that part of the issue was trying to satisfy that need.

**Comment by lodegaard** [comment]
It has been a problem for me since I'm dumping a lot of information when a test fails and the output in Jenkins became really messy. I mostly wanted to find out if anyone was interested in a fix
